flow-typed/npm/mapbox-gl_vx.x.x.js

```diff
@@ -0,0 +1,18 @@
+// flow-typed signature: 658d10f79c2258c2a56c07892d24093f
+// flow-typed version: <<STUB>>/mapbox-gl_v0.50.0/flow_v0.80.0
+
+/**
+ * This is an autogenerated libdef stub for:
+ *
+ *   'mapbox-gl'
+ *
+ * Fill this stub out by replacing all the `any` types.
+ *
+ * Once filled out, we encourage you to share your work with the
+ * community by sending a pull request to:
+ * https://github.com/flowtype/flow-typed
+ */
+
+declare module 'mapbox-gl' {
+  declare module.exports: any;
```

Comment:
Doesn't mapbox.gl export its flowtypes?

Reply:
Doesn't look like it https://github.com/mapbox/mapbox-gl-js/tree/master/build

Reply:
https://github.com/mapbox/mapbox-gl-js/pull/7079 looks like it'll be coming soon!